aip/general/0149.md

```diff
@@ -38,3 +38,29 @@ message Book {
 value and unset most of the time; if an alternative design does not require
 such a distinction, it is usually preferred. In practice, this means `optional`
 **should** only ever be used for integers and floats.
+
+**Important:** Tracking field presence is *not* the same as documenting API
+field behavior as defined in [AIP-203][]. For example, a field labeled with
+`optional` for presence tracking **may** also be annotated as
```

Comment:
Should there be a cross-reference to http://cloud/apis/design/design_patterns.md#optional_primitive_fields ?

Reply:
bah! Shouldn't *that* link to *this* AIP? Anyways, discussion of primitives, etc. is part of the guidance above, I don't think we need to discuss it further in this note. Is that fair?

Reply:
Yeah, I'd rather move more to the AIPs. it's more actively maintained.

The bifurcation cloud APIs and AIPs is a known issue though - when we have more breathing room we should probably tackle that.